internal/random/id.go

```diff
@@ -0,0 +1,37 @@
+package random
+
+import (
+	"errors"
+	"fmt"
+)
+
+// An Id represents the ID of an in-game entity. Its value is the integer representation of its original hex value.
+type Id struct {
```

Comment:
Is there a reason you prefer this struct over e.g. `type Id uint16`?

Reply:
I'm just used to writing wrappers, but as it's only a value the struct is unneccessary. I'm going to change that.